indra/cmake/UI.cmake

```diff
@@ -25,6 +25,8 @@ if (LINUX)
 
   target_link_libraries( ll::uilibraries INTERFACE
           fltk
+          pangocairo
+          wayland-client
```

Comment:
Wayland client is handled via pkgconfig (line 17). You can comment it in in line 20. It's commented out to find a better way, but for this draft there is no reason to not enable it

Reply:
Thank you.
The changes have been reflected.